client/src/components/Feed/NestedComments.js

```diff
@@ -1,32 +1,32 @@
 // Core
-import React, { useState } from "react";
+import React, {useState} from "react";
```

Comment:
It looks like this was formatted with an ESLint/Prettier config that is different from the one that we use. The Prettier config that we use maintains spaces around braces, and also uses 2 space indentation rather than 4. Can you check your ESLint/Prettier setup and see if you can reformat this file (and any other affected files) to use our configuration?

Reply:
Is there a way to do this at once for the entire project?

Reply:
So there is a husky pre-commit hook that is supposed to run Prettier and ESLint every time you commit:

![husky](http://g.recordit.co/uEg32SnhdU.gif)

I think it only works if you have installed husky, prettier, and eslint installed locally, but you're probably running the app in Docker this whole time. Can you try the following?
```
cd client
nvm use  # should switch to Node 12.16.2. If it is not installed, run nvm install 12.16.2
npm install --only=dev  # should install husky, prettier, lint-staged which is required for the husky pre-commit hook
```

And then make a change to one of the React files so that you can add a commit to run the husky pre-commit hook.

@joshmorel @idjevm Maybe we should add this to the Readme?

Reply:
@mannykary 

>  Maybe we should add this to the Readme?
> EDIT: Actually, we should probably do what you suggested in Slack this morning @joshmorel. I'll reach out to you sometime this week to discuss.

Yeah I think the root level option for linters/husky might be a good choice (or if not good at least the only sane choice). In the interim, I've been fixing these myself if I notice them in other PRs (certainly not ideal!) because right now we don't have any straight forward options if we ask people to use docker.

Reply:
Got it; @ashminbhandari if you are unable to fix the formatting issues, then I'll go ahead and do it before merging this branch to staging.